Support /etc/papersize

NB, this is now done in debian/rules, so it will automatically apply to
    every new wrapper script which gets added to this package in the
    future.  The patch should however not be deleted, so it does show
    up when looking at the differences with upstream.
--- foo2zjs_20100728.orig/foo2hp2600-wrapper.in
+++ foo2zjs_20100728/foo2hp2600-wrapper.in
@@ -204,7 +204,8 @@
 
 MEDIA=1
 COPIES=1
-PAPER=1
+test -r /etc/papersize && PAPER=$(cat /etc/papersize)
+test "$PAPER" || PAPER=1
 RES=600x600
 SOURCE=7
 NUP=
--- foo2zjs_20100728.orig/foo2oak-wrapper.in
+++ foo2zjs_20100728/foo2oak-wrapper.in
@@ -160,7 +160,8 @@
 QUALITY=wts
 MEDIA=1
 COPIES=1
-PAPER=1
+test -r /etc/papersize && PAPER=$(cat /etc/papersize)
+test "$PAPER" || PAPER=1
 RES=600x600
 SOURCE=7
 NUP=
--- foo2zjs_20100728.orig/foo2zjs-wrapper.in
+++ foo2zjs_20100728/foo2zjs-wrapper.in
@@ -194,7 +194,8 @@
 QUALITY=wts
 MEDIA=1
 COPIES=1
-PAPER=1
+test -r /etc/papersize && PAPER=$(cat /etc/papersize)
+test "$PAPER" || PAPER=1
 RES=1200x600
 SOURCE=7
 NUP=
